.ado/pipelines/config/loadtest-baseline-chaos.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "passFailMetrics": {
+            "ece127bc-6271-4a70-93f6-758c56a3f11c": {
```

Comment:
where do those guids(?) come from? self-generated?

Reply:
i've created these metrics via the portal and then extracted them from there.. i modified some of them manually and that works just fine.. i currently assume that they've to be unique per test..

Reply:
ok